content/intermediate/templates/resource-dependencies/index.md

```diff
@@ -247,9 +286,35 @@ S3Bucket:
       - Key: Name
         Value: Resource-dependencies-workshop
 ```
+
+:::::tabs{variant="container"}
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace/resource-dependencies`:
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  cd cfn101-workshop/code/workspace/resource-dependencies
+  :::
+1. Use the AWS CLI to create the stack. The required parameter `--template-body` have been pre-filled for you.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation create-stack --stack-name cfn-workshop-resource-dependencies-challenge --template-body file://resource-dependencies-with-intrinsic-functions.yaml
```

Comment:
Error due to wrong yaml file being specified: `An error occurred (ValidationError) when calling the CreateStack operation: Parameters: [EmailAddress] must have values`
Also added line break
```suggestion
  aws cloudformation create-stack --stack-name cfn-workshop-resource-dependencies-challenge \
  --template-body file://resource-dependencies-challenge.yaml
```